api/simple_authentication.py

```diff
@@ -28,7 +28,12 @@ class SimpleAuthenticationProvider(BasicAuthenticationProvider):
 
     ADDITIONAL_TEST_IDENTIFIERS = 'additional_test_identifiers'
 
-    SETTINGS = BasicAuthenticationProvider.SETTINGS + [
+    basic_settings = BasicAuthenticationProvider.SETTINGS
```

Comment:
Call `list(BasicAuthenticationProvider.SETTINGS)` to create a new list object -- otherwise you're modifying the list in place and it will affect every BasicAuthenticationProvider.